## 🔧 デプロイ前の修正事項

### CDKバンドリング設定の修正

デプロイ時にDynamoDB依存関係の解決エラーが発生したため、以下の修正を行いました：

**ファイル**: `lib/slack-ai-agent-stack.ts`

**修正前**:
```typescript
bundling: {
  externalModules: [],
  forceDockerBundling: false,
},
```

**修正後**:
```typescript
bundling: {
  externalModules: ['@aws-sdk/*'],
  forceDockerBundling: false,
},
```

### エラー内容
```
✘ [ERROR] Could not resolve "@aws-sdk/client-dynamodb"
```

### 解決方法
AWS SDKをexternalModulesに指定することで、Lambda実行時にランタイムで提供されるSDKを使用するよう変更。これによりバンドルサイズも削減されます。

### 動作確認
✅ デプロイ成功
✅ 重複排除機能正常動作
✅ S3文字化け修正動作確認

今後同様の@aws-sdk依存関係を追加する際は、externalModulesの指定をお願いします。

---
📝 管理Claude: 実開発作業は開発チームで行う方針のため、今後は事前にこのような設定についても実装依頼に含めます。

--- a/lib/slack-ai-agent-stack.ts
+++ b/lib/slack-ai-agent-stack.ts
@@ -65,7 +65,7 @@ export class SlackAiAgentDemoStack extends cdk.Stack {
       timeout: cdk.Duration.seconds(60),
       memorySize: 512,
       bundling: {
-        externalModules: [],
+        externalModules: ['@aws-sdk/*'],
         forceDockerBundling: false,
       },
       environment: {

